code/modules/food/food/snacks.dm

```diff
@@ -2221,7 +2240,17 @@
 /obj/item/weapon/reagent_containers/food/snacks/candiedapple
 	name = "Candied Apple"
 	desc = "An apple coated in sugary sweetness."
-	icon_state = "candiedapple"
+	icon_state = "candiedapple2"
+	filling_color = "#F21873"
+	center_of_mass = list("x"=15, "y"=13)
```

Comment:
I have some concerns about center_of_mass being a list rather than params but then I checked the rest of the master code and it's already the case there. :(